### DISCUSSION TITLE
About double-clicked Face-Up Stacks

### DISCUSSION THREAD

**Discussion by SexyNights** [finol-digital/Card-Game-Simulator#69] [Q&A] [answered]
Dear Finol Digital members.
I think that Your cgs is very good work. The fact that it can be operated with a smartphone is particularly astounding.
The operation of cgs is simple, but the necessary functions are implemented, and I use my PC and iPhone to learn/do experiment the cgs specifications.

However, I am having trouble with double-clicked Face-Up Stacks (Always reveal the top card).
I think that Clicking on the View icon is not a problem.
But During a versus game, players may double-click on stacks or decks.
If this happens, the hidden cards become visible and the game has to be aborted.
At the very least, it is sure to put a damper on entertainment.

So I would like to ask you a question.
How can I disable this function(double-clicked Face-Up Stacks) ?

I would really appreciate it if you could respond to this question.

**Comment by davidmfinol** [comment, marked as answer]
Thanks for kind words!

You raise a good point about double-clicking the stacks which I hadn't considered.
There currently isn't any way to disable this function, but I will add an option/toggle for it in the Play Settings, and I will try to include it in the next release version (should be in 2 weeks).

Thanks again for the feedback!

**Reply by SexyNights** [reply]
Thank you for your quick response :-)

I find the Face-Up Stacks feature useful for debugging when creating games. 
So I don't consider it unnecessary. The problem is the double-clicked behaviour.

Also, the table-top simulator is highly functional, but I think it is too highly functional and difficult to support smartphones.
When CGS grows up well, I think it has the potential to become a major tool for smartphones.

Thanks too for your quick response !

P.S.
I think it would be easier to use if the double-clicked dice could make a dice roll.
If you can afford it, please consider it.

**Reply by davidmfinol** [reply]
I just released v1.78, where I added "Double-Click to View Stacks" and "Double-Click to Roll Dice" to the Play Settings Menu:
![image](https://user-images.githubusercontent.com/1088474/194727935-238e8b89-fbd0-4002-80b1-19157ee275bf.png)

Please let me know if you were hoping for anything else!

**Reply by SexyNights** [reply]
Thank you very much for your quick response!
I will try it out as soon as possible.
I appreciate your response :-)

**Reply by SexyNights** [reply]
I have just checked the two operations.
**No problems** with both PC and iPhone.
Thank you very much !!!
